frontend/src/routes/groups/[group_id]/services/[service_id]/attributes/[attribute_id]/+page.svelte

```diff
@@ -0,0 +1,314 @@
+<script lang="ts">
+	import messages from '$media/messages.json';
+	import { page } from '$app/stores';
+	import {
+		createAttributeInputMap,
+		type AttributeInputMap
+	} from '$lib/components/CreateServiceRequest/ServiceDefinitionAttributes/shared';
+	import { useLibre311Context, useLibre311Service } from '$lib/context/Libre311Context';
+	import {
+		ASYNC_IN_PROGRESS,
+		asAsyncFailure,
+		type AsyncResult,
+		asAsyncSuccess
+	} from '$lib/services/http';
+	import { createInput, stringValidator, type FormInputValue } from '$lib/utils/validation';
+	import { Breadcrumbs, Button, Card, Input, Progress } from 'stwui';
+	import { fade, slide } from 'svelte/transition';
+	import type { EditServiceDefinitionAttributeParams } from '$lib/services/Libre311/Libre311';
+	import XMark from '$lib/components/Svg/outline/XMark.svelte';
+	import { goto } from '$app/navigation';
+
+	type AttributeEditValue = { key: string; name: string };
+
+	type AttributeEditInput = {
+		code: number;
+		required: boolean;
+		description: FormInputValue<string>;
+		dataTypeDescription: FormInputValue<string>;
+		values: AttributeEditValue[] | undefined;
+	};
+
+	const libre311 = useLibre311Service();
+	const alertError = useLibre311Context().alertError;
+
+	let asyncAttributeInputMap: AsyncResult<AttributeInputMap> = ASYNC_IN_PROGRESS;
+	let groupId = $page.params.group_id;
+	let serviceCode = Number($page.params.service_id);
+	let attributeCode = Number($page.params.attribute_id);
+	let groupName = '';
+	let serviceName = '';
+	let editAttributeInput: AttributeEditInput = {
+		code: 0,
+		required: false,
+		description: createInput<string>(),
+		dataTypeDescription: createInput<string>(),
+		values: undefined
+	};
+	let multivalueErrorMessage: string | undefined;
```

Comment:
Can we create an "EditMultivalueListAttribute" component and encapsulate all of the concerns related to updating and validating multivalue lists inside of there.

Otherwise this page will grow really large by the time we add all 6 attribute datatypes.